apiserver/facades/agent/uniter/uniter.go

```diff
@@ -2014,14 +2023,27 @@ func (u *UniterAPI) NetworkInfo(args params.NetworkInfoParams) (params.NetworkIn
 		info.IngressAddresses = bindingsToIngressAddresses[binding]
 
 		// If there is no ingress address explicitly defined for a given binding,
-		// set the ingress addresses to the binding addresses.
+		// set the ingress addresses to the either any defaults set above, or the binding addresses.
```

Comment:
I *think* "..to the either..." is not very English?... Don't you need to drop the "the"? :)